examples/svm/plot_svm_anova.py

```diff
@@ -19,21 +21,15 @@
 
 # #############################################################################
 # Import some data to play with
-X, y = load_digits(return_X_y=True)
-# Throw away data, to be in the curse of dimension settings
-X = X[:200]
-y = y[:200]
-n_samples = len(y)
-X = X.reshape((n_samples, -1))
-# add 200 non-informative features
-X = np.hstack((X, 2 * np.random.random((n_samples, 200))))
+X, y = load_iris(return_X_y=True)
+# Add non-informative features
+np.random.seed(0)
+X = np.hstack((X, 2 * np.random.random((X.shape[0], 36))))
 
 # #############################################################################
 # Create a feature-selection transform and an instance of SVM that we
 # combine together to have an full-blown estimator
-
 transform = SelectPercentile(chi2)
-
 clf = Pipeline([('anova', transform), ('svc', SVC(gamma="auto"))])
```

Comment:
```suggestion
clf = Pipeline([('anova', transform),
                ('scaler', StandardScaler()),
                ('svc', SVC(gamma="auto"))])

```

we can always normalize the data after feature selection. This gives a much better (IMO) plot as well.
![figure_1](https://user-images.githubusercontent.com/1663864/51790669-cc6dbc00-2198-11e9-97d3-54e743568e9b.png)